frontend/components/alert.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * A simple alert component that creates an alert and then dismisses it after 2s
+ * @param {Element} parent
+ * @param {string} type
+ * @param {string} message
+ * @param {boolean} autohide - NYI
+ * @returns
+ */
+export function createAlert (parent, type, message, autohide) {
```

Comment:
This is a neat little component, try to enter wring PIN 3 times and you can see it in action. Could have more use cases maybe? Makes the app fell a bit more alive imo 😄